client/src/main/python/slipstream/executors/MachineExecutor.py

```diff
@@ -27,6 +27,7 @@
     TerminalStateException, ExecutionException
 from slipstream import util
 from slipstream.Client import Client
+from slipstream.wrappers.BaseWrapper import InconsistentScaleStateError
```

Comment:
After migrating scalability related exceptions to `Exceptions` module, this import doesn't fail in the unit tests, because `slipstream.wrappers.BaseWrapper` module by itself imports `InconsistentScaleStateError`.  
Anyways, I'm still going to make the proper import here from `Exceptions`.